library/psa_crypto_mac.c

```diff
@@ -401,8 +391,8 @@ static psa_status_t mac_sign_setup(
     size_t key_buffer_size,
     psa_algorithm_t alg )
 {
-    return( mac_setup( operation, attributes, key_buffer, key_buffer_size, alg,
-                       1 ) );
+    return( mac_setup( operation,
+                       attributes, key_buffer, key_buffer_size, alg ) );
 }
 
 static psa_status_t mac_verify_setup(
```

Comment:
`mac_verify_setup` and `mac_sign_setup` are now identical. Why not call `mac_setup` directly?

Reply:
done